core/src/scopyhomepage.cpp

```diff
@@ -71,6 +72,16 @@ void ScopyHomePage::disconnectDevice(QString id) {
 	db->disconnectDevice(id);
 }
 
+void ScopyHomePage::disconnectEmuDevice(QString id)
```

Comment:
I don't think we should do this. We use the emulator for development. We can use it to simulate loss of connection with the board by disabling the emulator. 
Therefore the plugins should gracefully handle this, and not be handled by the home page (by forcefully disconnecting the device)